visualization/app/codeCharta/ui/toolBar/toolBar.component.scss

```diff
@@ -34,7 +34,12 @@ tool-bar-component {
 		right: 10px;
 		display: inline-flex;
 		align-items: center;
-		width: 150px;
+		width: 400px;
```

Comment:
would it be possible to not hard code this size and increase the responsability?

Reply:
we could use % instead of px